Add governance jobs (#280)
kyma-project/test-infra: prow/scripts/README.md

```
  ├── cluster-integration                # Scripts for executing the multi-step integration job on a Google Kubernetes Engine (GKE) cluster. This job provisions a cluster, sets up a DNS record, an IP address, and a TLS certificate. It also installs Kyma on the cluster and performs tests on it.
  ├── build.sh                           # This script builds and tests a given Kyma component by running the respective "Makefile" target.

>   ├── governance.sh                      # This script runs milv bot for validation internal and external links in markdown files. On PRs checks all internals and externals links on changed markdown files. Also checks all links once a day on master.

Reviewer: ```suggestion
  ├── governance.sh                      # This script runs the "milv" bot for validating internal and external links in Markdown files. It checks all internal and external links in ".md" files changed in PRs. It also checks all links on the master branch once a day.
```

>   ├── library.sh                         # This script is used as an integral part of other scripts, for example by the "build.sh" script. With proper parameters defined, it authenticates you to GCP and sets up the Docker-in-Docker environment.

Reviewer: ```suggestion
  ├── library.sh                         # This script is used as an integral part of other scripts, such as the "build.sh" script. With proper parameters defined, it authenticates you to GCP and sets up the Docker-in-Docker environment.
```
